charts/azure-metrics-exporter/Chart.yaml

```diff
@@ -3,13 +3,13 @@ name: azure-metrics-exporter
 type: application
 description: A Helm chart for azure-metrics-exporter
 home: https://github.com/webdevops/azure-metrics-exporter
-version: 1.0.4
+version: 1.0.5
```

Comment:
As this is a breaking change in the helm chart, I would suggest to bump the major version.

And maybe start a simple changelog file with something like "Defaults for a metricsProbe are now under `probeDefaults`. Move everything under `prometheus.metricProbes` to `prometheus.metricProbes.probeDefaults`, except for `prometheus.metricProbes.probes`"

Apart from that: looks good to me, but I'm just an user :)